ak.py

```diff
@@ -89,6 +128,19 @@ def main(self, *args):
         return self.parent.log_and_exec(command, params, local.env)
 
 
+@Ak.subcommand("upgrade")
+class AkUpgrade(cli.Application, DbTools):
+    """Upgrade odoo."""
+
+    def main(self, *args):
+        db = self.determine_db()
+        params = []
+        params += ['-d', db]
+        command = 'bin/upgrade_openerp'
```

Comment:
Hi maybe I miss something but when you run bin/upgrade_openerp the script "upgrade_openerp" will already read the config file. We only have to pass a new db if we want to force manually a specific db